src/assert.js

```diff
@@ -173,6 +173,38 @@ class Assert {
 		} );
 	}
 
+	true( result, message ) {
+		const isTrue = result === true;
+		if ( !message ) {
+			message = isTrue ?
+				"true" :
+				`failed, expected argument to be true, was: ${dump.parse( result )}`;
```

Comment:
I haven't run this yet to see what the output looks like in the HTML and CLI/Tap reporter, but I suspect this is redundant given we don't do this for equal etc either. I believe `ok()` did it mainly because it's comparison target was technically unknown, so we did not want to put too much emphasis in the actual/expected part where it would say `true` given that `ok()` also allows e.g. random objects and functions as truthy. But this does not apply to assert.true/false, I'd expect them to behave the same as `strictEqual(, true/false)`.

Reply:
Makes sense. I removed the custom message. See a screenshot below and let me know if that works for you.

<img width="647" alt="output" src="https://user-images.githubusercontent.com/5890858/85360873-465dbe00-b4cf-11ea-8e0d-60334f84ef7a.png">